src/ansys_sphinx_theme/__init__.py

```diff
@@ -474,7 +478,76 @@ def update_search_sidebar_context(
     context["sidebars"] = sidebar
 
 
-def setup(app: Sphinx) -> Dict:
+def on_doctree_resolved(app: Sphinx, doctree: nodes.document, docname: str) -> None:
+    """Add a 'Home' entry to the root TOC.
+
+    Parameters
+    ----------
+    app : Sphinx
+        Sphinx application instance for rendering the documentation.
+    doctree : nodes.document
+        Document tree for the page.
+    docname : str
+        Name of the current document.
+
+    Notes
+    -----
+    This function checks if the 'Home' entry already exists in the root TOC.
+    If it does not exist, it adds the 'Home' entry at the beginning of the TOC.
+    The 'Home' entry links to the index page of the documentation.
+    """
+    index_page = "index"
+    root_toc = app.env.tocs[app.config.root_doc]
+    for toc in traverse_or_findall(root_toc, toctree):
+        if not toc.attributes.get("entries"):
+            return
+
+        for title, page in toc.attributes["entries"]:
+            if title == "Home":
+                return
+
+        home_entry = (
+            nodes.Text("Home"),
+            index_page if index_page != docname else None,
+        )
+        # Insert 'Home' entry at the beginning of the TOC
+        toc.attributes["entries"].insert(0, home_entry)
+
+
+def add_tooltip_after_build(app: Sphinx, exception):
+    """Add tooltips to 'Home' links after the build process.
+
+    Parameters
+    ----------
+    app : Sphinx
+        Sphinx application instance for rendering the documentation.
+    exception : Exception
+        Exception raised during the build process.
+
+    Returns
+    -------
+    None
+    """
+    if exception:
+        return
+
+    outdir = pathlib.Path(app.outdir)
+    project_name = f"{app.config.html_short_title} home" or None
+    if not project_name:
+        project_name = f"{app.config.project} home" or "Package Home"
+
+    for html_file in outdir.rglob("*.html"):
+        with html_file.open("r", encoding="utf-8") as f:
```

Comment:
Avoid using single letter variables:


```suggestion
        with html_file.open("r", encoding="utf-8") as file:
```